test/fixtures/linter/rules/NoDeprecatedApi/ManagedObject.js

```diff
@@ -0,0 +1,78 @@
+sap.ui.define(["sap/ui/core/Control", "sap/m/library", "mycustom/lib/FancyMultiPage"], 
```

Comment:
The file is called ManagedObject, but the class inherits from Control. I think it makes sense to test both: A sub-class of ManagedObject and a sub-class of Control.

Reply:
Done